configuration/driver/icedrv_restart.F90

```diff
@@ -339,7 +339,8 @@ subroutine read_restart_field(nu,work,ndim)
 
       minw = minval(work)
       maxw = maxval(work)
-      write(nu_diag,*) minw, maxw
+      sumw = sum(work)
+      write(nu_diag,*) subname, minw, maxw, sumw
       
```

Comment:
Do you still want the print statements here?

Reply:
They were there before, and I've found them to be handy.

Reply:
These were there before and I just extended the feature a bit.  One of the first things I wanted to check was whether all the fields were read as written and whether they were identical.  It's helpful to have the same diagnostics written when writing a restart as when reading it, and the sum is a new diagnostic that provides something like a checksum value of the field.